biojava-core/src/test/resources/uppercase_locus0.gb

```diff
@@ -0,0 +1,7 @@
+LOCUS       ABC12.3_DE   7071 BP DS-DNA   CIRCULAR  SYN       22-JUL-1994
```

Comment:
Can you place all these files under the same resource directory as the testing class? We haven't always followed that convention but we are trying to do it going forward.